dist/rpm/singularity.spec.in

```diff
@@ -45,130 +45,65 @@ Requires: squashfs
 Requires: squashfs-tools
 %endif
 
-Requires: %{name}-runtime = %{version}-%{release}
+
+Requires: %{name}-unpriv = %{version}-%{release}
+Provides: %{name}-runtime
+Obsoletes: %{name}-runtime
```

Comment:
Is this right? You're saying it `Provides` ${name}-runtime then `Obsoletes` ${name}-runtime?

I can see the `Obsoletes` here... but not the `Provides`. :/

Reply:
Yes. that's right.  If someone currently has installed only singularity-runtime and they do a yum upgrade I want them to get the new singularity.  The purpose of singularity-runtime was to minimize exposure to setuid while still allowing a complete runtime environment, without necessarily including the ability to create images.  @bauerm97  told me that there's no privileged functionality used in 3.0 for creating images, so it doesn't hurt to include that functionality and there's no purpose for a separate singularity-runtime package anymore.